### PR TITLE
[CONSVC-1709] chore: enable tests and coverage for circle

### LINKED ISSUE #15: Set up circle to run tests
Currently the circleci config is set so that there is a migration check and lining.

We would want it to check for tests passing too.
This will require docker user and password env vars? and introducing postgres into the config

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/CONSVC-1709)

### PR DESCRIPTION
**Pull request by tiftran**
Enables tests for circle.
Docker credentials are temp, since it's current using mine

closes #15

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -33,10 +42,18 @@ jobs:
...
+            if [ "${DOCKER_USER_TEMP}" == "" ] || [ "${DOCKER_PASS_TEMP}" == "" ]; then
+              echo "Skipping Login to Dockerhub, credentials not available."
+            else
+              echo "${DOCKER_PASS_TEMP}" | docker login -u="${DOCKER_USER_TEMP}" --password-stdin
+            fi
+      - run:
+          name: Run tests
           # This assumes pytest is installed via the install-package step above
-      #    command: pytest
+          command: pytest --cov --cov-report=term-missing --cov-fail-under=95
```
Comment (ncloudioj) [inline .circleci/config.yml:56]: How about `--cov-fail-under=100`? ;)

Comment (ddurst) [reply .circleci/config.yml:56]: 😍

Comment (ddurst) [reply .circleci/config.yml:56]: (I am kidding)

Comment (tiftran) [reply .circleci/config.yml:56]: 😆

Comment (tiftran) [reply .circleci/config.yml:56]: 95 wasn't arbitrary 🙂. With #36 it's at 95-ish%, lines that weren't covered were the `__str__` methods, which I decided probs weren't worth the trouble

**Comment by tiftran** [comment]
merging with $DOCKER_USER_TEMP and $DOCKER_PASS_TEMP. Will create issue to use real credentials once they're set

**Pull request merged**